chef_master/source/server_security.rst

```diff
@@ -310,6 +310,111 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
+
+   .. code-block:: bash
+   
+      $ /path/to/postgresql/postgresql restart
+
+#. Edit ``/etc/opscode/chef-server.rb`` on the Chef Infra Server machine and add the following line:
+
+   .. code-block:: ruby
+   
+      postgresql['sslmode']='require'
+
+#. Run reconfigure on the Chef Infra Server machine:
+
+   .. code-block:: bash
+   
+      $ chef-server-ctl reconfigure
+
+#. Verify that SSL is enabled and that SSL connections are up between Chef Infra Server and your running PostgreSQL instance.  One way to do this is by running ``chef-server-ctl psql`` on the Chef Infra Server machine and confirming that most connections in the ssl column are marked 't' (true):
+
+   .. code-block:: bash
+   
+      $ chef-server-ctl psql opscode_chef 
+   
+      postgres=# show ssl;
+       ssl
+      -----
+       on
+      (1 row)
+   
+      postgres=# select * from pg_stat_ssl;
+   
+        pid  | ssl | version |           cipher            | bits | compression | clientdn
+      -------+-----+---------+-----------------------------+------+-------------+----------
+       16083 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16084 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16085 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16086 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16087 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16088 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16089 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16090 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16091 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16092 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16093 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16094 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16095 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16096 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16097 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16098 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16099 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16100 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16101 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16102 | t   | TLSv1.2 | ECDHE-RSA-AES256-GCM-SHA384 |  256 | f           |
+       16119 | f   |         |                             |      |             |
+      (21 rows)
+
+      postgres=# \q
```

Comment:
We probably don't need to tell people to quit their session.
```suggestion
```

Reply:
Done.